pyproject.toml

```diff
@@ -40,6 +40,8 @@ examples = ["transformers", "datasets", "scipy", "torch", "pandas"]
 api = [
     "fastapi[standard]",
     "uvicorn",
+    "streamlit",
+    "websockets"
```

Comment:
can we use AIOHTTP instead? therefore we don't need new package

Reply:
I am not sure about this. When I tried to host the websocket api it prompts me "No supported WebSocket library detected. Please use "pip install 'uvicorn[standard]'", or install 'websockets' or 'wsproto' manually."

Reply:
okay, let's not worry about that for now